annotate.gemspec

```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |s|
   s.name = "annotate"
   s.version = Annotate.version
 
-  s.required_ruby_version = '>= 1.9.3'
+  s.required_ruby_version = '>= 2.2.5'
```

Comment:
Jumping all the way to 2.2.5 seems a little overkill.  The 2.1 series is still receiving security patches, so in my opinion it should probably stay supported unless there is an overwhelming benefit of dropping it.